torch/_functorch/aot_autograd.py

```diff
@@ -423,9 +423,90 @@
 aot_autograd_decompositions = {}
 
 
+FakifiedFlatArgs = NewType("FakifiedFlatArgs", List[Any])
+
+
+def process_inputs(
```

Comment:
I think it is but just confirming that this is code motion?

Reply:
Do you mean this is just moving stuff outside so it can be called by aot_module_simplified, aot_function etc.